src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx

```diff
@@ -35,21 +38,12 @@ export const TranscriptWidget = ({
   allowTranscriptDownloads,
   showTranscriptByDefault,
   updateField,
+  isUploadError,
+  isDeleteError,
 }) => {
   const languagesArr = hooks.transcriptLanguages(transcripts);
-  const fileInput = hooks.fileInput();
-  const input = {
-    error: {
-      dismiss: () => { console.log('dismiss'); },
-      show: true,
-    },
-  };
-  const upload = {
-    error: {
-      dismiss: () => { console.log('dismiss'); },
-      show: true,
-    },
-  };
+  const fileInput = hooks.fileInput({ onAddFile: '' });
```

Comment:
We'll need to add an onAddFile action here in order to integrate our work

Reply:
Wait, now that I've read the file, can we follow the same pattern of passing in an action for both, as not to confuse the reader of our code?